services/QuillLMS/Rakefile

```diff
@@ -2,7 +2,30 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require File.expand_path('../config/application', __FILE__)
+require 'dotenv/tasks'
 
 EmpiricalGrammar::Application.load_tasks
 
 task default: [:spec]
+
+task after_hook: :dotenv do
+  post_slack_rake_task
+end
+
+Rake.application.tasks.reject do |task|
```

Comment:
There's a few rake tasks I think we'd want to ignore:

1. Asset compilation during deploy, e.g. `rake assets:precompile`, `rake assets:clean`, [our enhance method](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/lib/tasks/assets.rake#L7). Note, you used these in the PR example, so you might be thinking differently. Happy to discuss.
2. [Our generic cron rake tasks](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/lib/tasks/cron.rake#L1)

Scanning `lib/tasks`, I think that's it (but if others pop up, we can add them later).

Reply:
While testing this on staging, I used `db:migrate:status`  and `db:rollback STEP=0`.  The `rake assets:precompile`, `rake assets:clean` fired errantly, so I would agree that we need these on an ignore list.
![Screen Shot 2021-11-15 at 1 28 14 PM](https://user-images.githubusercontent.com/2057805/141834966-bd13c06d-adf0-4e88-8d95-bb4028f4ed2f.png)